core/src/services/routing.js

```diff
@@ -44,10 +44,13 @@ class RoutingClass {
   /**
     navigateTo used for navigation
     Triggers a frame reload if we are on the same route (eg. if we click on same navigation item again)
-    @param route string  absolute path of the new route
+    @param {string} route absolute path of the new route
+    @param {Object} options navigation options of keepBrowserHistory, navSync, preventContextUpdate
```

Comment:
Could you add here something similar to [this example parameter](https://github.com/SAP/luigi/blob/master/core/src/core-api/ux.js#L31) or [this example parameter](https://github.com/SAP/luigi/blob/master/core/src/core-api/ux.js#L31)

I would assume it would like like this:

```
@param {Object} options navigation options
@param {Object} options.keepBrowserHistory <<explanation here>>
@param {Object} options.navSync <<explanation here>>
@param {Object} options.preventContextUpdate <<explanation here>>
```

It would be pretty helpful when debugging something in the future.